src/main/java/lk/sack/livescore/api/MainController.java

```diff
@@ -63,11 +63,13 @@ public void updateScore(HttpSession session, @RequestBody TestMatch match) throw
     }
 
     @PostMapping("/user-status")
-    public void checkUserStatus(HttpSession session) throws UnauthorizedUserException {
+    public String checkUserStatus(HttpSession session) throws UnauthorizedUserException {
         if (session.getAttribute("isAuthorizedUser") == null || session.getAttribute("isAuthorizedUser").equals(false)) {
-            String message = "Unauthorized attempt to update the score";
+            String message = "Unauthorized attempt to access score update dashboard";
```

Comment:
This message depends on the UI. Shall we use a generic message like "unauthorized user"?

Reply:
Okay